vmdb/app/helpers/js_helper.rb

```diff
@@ -61,6 +61,10 @@ def javascript_del_class(element, cls)
     "$j('##{j_str(element)}').removeClass('#{j_str(cls)}');".html_safe
   end
 
+  def javascript_disable_field(element, status)
+    "$j('##{j_str(element)}').prop('disabled', #{status});".html_safe
```

Comment:
@h-kataria Let's create 2 functions, javascript_enable_field and javascript_disable_field that only take the element parameter, as passing true/false to the "disable" function is a bit confusing and it will be clearer this way.